addons/gamification/models/gamification_karma_rank.py

```diff
@@ -17,11 +17,30 @@ class KarmaRank(models.Model):
     karma_min = fields.Integer(string='Required Karma', required=True, default=1,
         help='Minimum karma needed to reach this rank')
     user_ids = fields.One2many('res.users', 'rank_id', string='Users', help="Users having this rank")
+    rank_users_count = fields.Integer("Nb. Users", compute="_compute_rank_users_count")
 
     _sql_constraints = [
         ('karma_min_check', "CHECK( karma_min > 0 )", 'The required karma has to be above 0.')
     ]
 
+    @api.depends('user_ids')
+    def _compute_rank_users_count(self):
+        requests_data = self.env['res.users'].read_group([], ['rank_id'], ['rank_id'])
+        requests_mapped_data = dict((data['rank_id'][0], data['rank_id_count']) for data in requests_data)
+        for rank in self:
+            rank.rank_users_count = requests_mapped_data.get(rank.id, 0)
+
+    def current_rank_users(self):
```

Comment:
as explain in a previous review, create these ir.actions.act_window in xml with default value on current id.
instead to use python method...